.gitignore

```
@@ -1,3 +1,4 @@
+venv/*
```

Should be
```suggestion
venv/
```
or maybe even
```suggestion
/venv/
```
if we place it in the root in our docs.

---

Sure, sounds good. Do you want the / at the end of it? I would guess `/venv` would do the same job?

---

I like to keep the trailing `/` because that unambiguously indicates that it's for a directory.

---

sounds good. Done. :)